src/config/dynamic-theme-fixes.config

```diff
@@ -10063,6 +10063,19 @@ CSS
 
 ================================
 
+infinity-academies.com
+
+INVERT
+img.bb-logo
+img[src*="profile-avatar-buddyboss-50.png"]
+
+CSS
+.wp-image-547 {
+    border-radius: 15px !important;
+}
```

Comment:
This is a cosmetic fix, those aren't allowed in the sites fixes.

Reply:
Oh that's sad